api/v1alpha1/generator_options.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 Red Hat, Inc.
```

Comment:
Also while we are at it, could you do a Cmd-F and search if the copyright years are correct and update it

Reply:
Done.  It was just that file that needed to be updated